.gitignore

```diff
@@ -40,3 +40,4 @@ plugins-prod
 sandbox
 wave-tests
 x/*
+.DS_Store
```

Comment:
This gets generated on my mac so I thought I would add it

Reply:
Please do not include this in the PR